src/main/scala/io/iohk/ethereum/jsonrpc/EthBlocksService.scala

```diff
@@ -0,0 +1,198 @@
+package io.iohk.ethereum.jsonrpc
+
+import akka.util.ByteString
+import io.iohk.ethereum.domain.Blockchain
+import io.iohk.ethereum.jsonrpc.EthService.{BlockParam, ResolvedBlock}
+import io.iohk.ethereum.ledger.Ledger
+import io.iohk.ethereum.utils.{BlockchainConfig, Logger}
+import monix.eval.Task
+import org.bouncycastle.util.encoders.Hex
+
+object EthBlocksService {
+  case class ChainIdRequest()
```

Comment:
Yes, it can. But such a change demands adding more boilerplate code, where `ChainIdRequest` is being referred as the type of an object we have to explicitly replace `ChainIdRequest` by `ChainIdRequest.type`.